connector/src/test/scala/com/microsoft/kusto/spark/KustoSourceE2E.scala

```diff
@@ -106,9 +107,9 @@ class KustoSourceE2E extends FlatSpec with BeforeAndAfterAll {
       .option(KustoSinkOptions.KUSTO_CLUSTER, cluster)
       .option(KustoSinkOptions.KUSTO_DATABASE, database)
       .option(KustoSinkOptions.KUSTO_TABLE, table)
-      .option(KustoSinkOptions.KUSTO_AAD_APP_ID, appId)
-      .option(KustoSinkOptions.KUSTO_AAD_APP_SECRET, appKey)
-      .option(KustoSinkOptions.KUSTO_AAD_AUTHORITY_ID, authority)
+//      .option(KustoSinkOptions.KUSTO_AAD_APP_ID, appId)
```

Comment:
I assume you commented these out to test the Device Code flow, but since you didn't push changes to the test to use DC, maybe these 3 should be left uncommented in the code.

Reply:
ill revert this file